staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go

```diff
@@ -289,6 +289,11 @@ const (
 	NonStructuralSchema CustomResourceDefinitionConditionType = "NonStructuralSchema"
 	// Terminating means that the CustomResourceDefinition has been deleted and is cleaning up.
 	Terminating CustomResourceDefinitionConditionType = "Terminating"
+	// KubeAPIApproved indicates that an API in *.k8s.io or *.kubernetes.io is or is not approved.  For CRDs
```

Comment:
we omit the k8s.io and kubernetes.io cases intentionally? Unimportant edge case?

Reply:
> we omit the k8s.io and kubernetes.io cases intentionally? Unimportant edge case?

The code restricts it.  I figured this was simpler for the docs.